addons/cba_settings/cba_settings.sqf

```diff
@@ -25,10 +25,13 @@ ace_hearing_enabledForZeusUnits = false;
 
 ace_interaction_disableNegativeRating = true;
 
-ace_map_defaultChannel = 1;
+ace_map_defaultChannel = 0; // Global (default: -1)
```

Comment:
We had this on `1` so people don't put markers into Global by default.

Reply:
But now we use Global as default.